components/shortcut/shortcut.app.js

```diff
@@ -0,0 +1,113 @@
+const { ShortcutClient } = require("@useshortcut/client");
+const get = require("lodash/get");
+const retry = require("async-retry");
+
+module.exports = {
+  type: "app",
+  app: "shortcut",
+  methods: {
+    api() {
+      return new ShortcutClient(this.$auth.api_key);
+    },
+    _isRetriableStatusCode(statusCode) {
+      [
+        408,
+        429,
+        500,
+      ].includes(statusCode);
+    },
```

Comment:
This function needs to return the validation with the statement `return` right?

Reply:
You are right))